orchestrator/modules/tools/discovery/handlers_playbooks.py

```
@@ -481,6 +481,45 @@ async def execute_playbook(db: Session, workspace_id: UUID, params: Dict[str, An
+    # never poison the launch transaction.
+    try:
+        from modules.tools.discovery.handlers_watches import auto_create_watch
+
+        criteria = f"Playbook '{playbook.name}' completes and delivers its expected output."
+        if input_data:
+            import json as _json
+
+            try:
+                criteria += f" Inputs: {_json.dumps(input_data, default=str)[:400]}"
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)